src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs

```diff
@@ -9,9 +25,17 @@ namespace CommunityToolkit.Maui.Core.Views;
 /// </summary>
 public class MauiMediaElement : Grid, IDisposable
 {
+	bool navbarExist;
+	bool isDisposed;
+
+	static readonly AppWindow? appWindow = GetAppWindowForCurrentWindow();
 	readonly MediaPlayerElement mediaPlayerElement;
 
-	bool isDisposed;
+	ImageSource source { get; set; } = new BitmapImage(new Uri("ms-appx:///whitefs.png"));
+	Button btn { get; set; }
+	Grid grid { get; set; } = new();
+	Grid ButtonContainer { get; set; }
+	Popup popup { get; set; } = new();
```

Comment:
those can be fields, since they are private

Reply:
The properties changed to fields as recommended. Thanks @pictos . Marked as resolved.